impeller/renderer/blit_pass.cc

```diff
@@ -48,8 +48,8 @@ bool BlitPass::AddCopy(std::shared_ptr<Texture> source,
     VALIDATION_LOG << SPrintF(
         "The source sample count (%d) must match the destination sample count "
         "(%d) for blits.",
-        source->GetTextureDescriptor().sample_count,
-        destination->GetTextureDescriptor().sample_count);
+        static_cast<int>(source->GetTextureDescriptor().sample_count),
+        static_cast<int>(destination->GetTextureDescriptor().sample_count));
```

Comment:
Nit: static_cast<std::underlying_type<SampleCount>>?

Reply:
I guess it makes sense to always interpret as int here. The cast should fail if it were even possible for something exotic to happen (won't happen).

Reply:
This should be `int`, since `%d` placeholders require an `int` argument.